### PR TITLE
reduce nesting where possible

### PR DESCRIPTION
**Pull request by MichaelChirico**
Uncovered by #2302

### CODE REVIEW

File: inst/example/complexity.R [hunk 1]
```diff
@@ -3,6 +3,7 @@ complexity <- function(x) {
     if (x > 10.0) {
       if (x > 20.0) {
         x <- x / 2.0
+        return(x)
```
Comment (MichaelChirico) [inline inst/example/complexity.R:6]: maybe it's preferable to delete this file? Why do we keep it here... it's used in two tests, shouldn't we prefer to write out the code directly in those tests?

Comment (IndrajeetPatil) [reply inst/example/complexity.R:6]: Yes, I agree. We can either directly include this code in tests or move this file somewhere to the `tests` folder and use it from there.

Comment (AshesITR) [reply inst/example/complexity.R:6]: I would vote for ditching the file.
We can also lower the cyclocomp threshold in the tests to make the test code smaller.

Comment (MichaelChirico) [reply inst/example/complexity.R]: Reminded me I forgot to push the edit I made locally

**Review by IndrajeetPatil — APPROVED** [review]

**Pull request merged**